runtime/precompiles/src/StakingInterface.sol

```diff
@@ -60,26 +59,18 @@ interface ParachainStaking {
 // https://ethereum.stackexchange.com/a/73405/9963
 // Eventually we will probably want a better way of generating these and copying them to Rust
 // {
-// 	"42a72461": "at_stake(uint256,address)",
 // 	"289b6ba7": "candidate_bond_less(uint256)",
 // 	"c57bd3a8": "candidate_bond_more(uint256)",
-// 	"96b41b5b": "candidate_pool()",
-// 	"b0c0081f": "collator_commission()",
-// 	"427c3c14": "exit_queue()",
```

Comment:
We shouldn't be removing these. If they aren't implemented, we should have followups. Actually having them in the code is more important than here. The code is what is used to generate these signatures.

Reply:
Why should we have them in the code if they're not implemented?  When we remove them, it doesn't change the other signatures. Each signature is generated independently.

Should I add the other storage items that aren't included here but have storage items in parachain-staking as well (ie Total, NominatorState, CollatorState, AwardedPts)?

All the followups are tracked in jira 495

Reply:
I'm just going to revert the file to the state you left it in and add min_nomination. I guess we also need to add min_nominator_stk too because that's another one and we should have all the config associated types that aren't storage items.